packages/mobile/src/components/lineup-tile/LineupTileStats.tsx

```diff
@@ -128,68 +134,72 @@ export const LineupTileStats = ({
   }, [dispatch, id, navigation, repostType])
 
   const downloadStatusIndicator = isCollection ? (
-    <CollectionDownloadStatusIndicator size={18} collectionId={id} />
+    <CollectionDownloadStatusIndicator size={spacing(4)} collectionId={id} />
   ) : (
-    <TrackDownloadStatusIndicator size={18} trackId={id} />
+    <TrackDownloadStatusIndicator size={spacing(4)} trackId={id} />
   )
 
   const isReadonly = variant === 'readonly'
 
   return (
-    <View style={styles.stats}>
-      {isTrending ? (
-        <LineupTileRankIcon showCrown={showRankIcon} index={index} />
-      ) : null}
-      {premiumConditions ? (
-        <LineupTilePremiumContentTypeTag
-          premiumConditions={premiumConditions}
-          doesUserHaveAccess={doesUserHaveAccess}
-          isOwner={isOwner}
-        />
-      ) : null}
-      {hasEngagement && !isUnlisted && (
-        <View style={styles.leftStats}>
-          <TouchableOpacity
-            style={[
-              trackTileStyles.statItem,
-              !repostCount ? styles.disabledStatItem : null
-            ]}
-            disabled={!repostCount || isReadonly}
-            onPress={handlePressReposts}
-          >
-            <Text style={trackTileStyles.statText}>
-              {formatCount(repostCount)}
-            </Text>
-            <IconRepost
-              height={16}
-              width={16}
-              fill={neutralLight4}
-              style={[styles.statIcon, styles.repostStat]}
-            />
-          </TouchableOpacity>
-          <TouchableOpacity
-            style={[
-              trackTileStyles.statItem,
-              !saveCount ? styles.disabledStatItem : null
-            ]}
-            disabled={!saveCount || isReadonly}
-            onPress={handlePressFavorites}
-          >
-            <Text style={trackTileStyles.statText}>
-              {formatCount(saveCount)}
-            </Text>
-            <IconHeart
-              style={[styles.statIcon, styles.favoriteStat]}
-              height={14}
-              width={14}
-              fill={neutralLight4}
-            />
-          </TouchableOpacity>
-          <View style={[trackTileStyles.statItem]}>
-            {downloadStatusIndicator}
+    <View style={styles.root}>
```

Comment:
This is mostly just wrapping in another `View`. Also changed `16` to `spacing(4)`.